packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -364,14 +406,24 @@ class _ModalScopeState extends State<_ModalScope> {
   }
 }
 
+/// Where a [ModalRoute] should be positioned within the [Navigator]'s [Overlay].
 class ModalPosition {
```

Comment:
We really should use RelativeRect here. It's exactly the same data model but has sprouted more helper functions.